package.json

```diff
@@ -10,10 +10,25 @@
     "url": "https://github.com/os-scar/overlay/issues",
     "email": "contact@os-scar.com"
   },
+  "scripts": {
+    "postinstall": "husky install",
+    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
+    "test:integration": "node --experimental-vm-modules node_modules/jest/bin/jest.js tests",
+    "build": "gulp build",
+    "build:watch": "gulp watch",
+    "lint": "eslint src tests",
+    "lint:fix": "yarn lint --fix",
+    "lint:lockfile": "lockfile-lint --path yarn.lock --allowed-hosts npm yarn --validate-https --validate-package-names --validate-integrity --empty-hostname false",
+    "lint:firefox": "yarn build && addons-linter dist/firefox",
+    "__start": "web-ext run --devtools --start-url https://stackoverflow.com/questions/33527653",
+    "start:chrome": "yarn __start --source-dir ./dist/chrome --target chromium",
+    "start:firefox": "yarn __start --source-dir ./dist/firefox"
```

Comment:
You can watch the `dist` files for both *Chrome* and *Firefox*.

Combined with `build:watch`, it should reload the extension every time you change the code.